server/routes/alumni.js

```diff
@@ -47,15 +47,28 @@ router.post('/', function (req, res, next) {
   var newAlumni = new alumni(req.body.newAlumni);
   var errors = [];
 
+  function validateUrls() {
+    var userGitHub = newAlumni.github
+    var re = /^(?:http(s)?:\/\/)?[\w.-]+(?:\.[\w\.-]+)+[\w\-\._~:/?#[\]@!\$&'\(\)\*\+,;=.]+$/
```

Comment:
So this checks to make sure we have a valid url, but it may be worth actually verifying the url contains "github" as well

Reply:
I should also show you how to test code at this point